lib/item/download_items_helper.rb

```diff
@@ -480,6 +483,17 @@ def generate_json_log(valids, invalids)
       json_log.to_json
     end
 
+    def filter_item_files(filenames, regexp_filter=DEFAULT_DOCUMENT_FILTER)
```

Comment:
Given a long array of various filenames that you have created in your test I'd be checking what comes out is what is expected.

e.g.:
filenames=['cat.cat','foo','']
asserting: *cat *.cat cat* cat.* food* catcat* etc. etc. etc.
Much easier to test at the method level than it would be at the controller level.